### ISSUE TITLE
Error	XFC0000	Cannot resolve type "clr-namespace:FFImageLoading.Transformations;assembly=FFImageLoading.Maui:fftransformations:GrayscaleTransformation"

### ISSUE THREAD

**Issue by SandipAhluwalia** [microspaze/FFImageLoading.Maui#33]
### Description

Code with GrayscaleTransformation not building

### Steps to Reproduce

Key part of project file looks like:

```
<PropertyGroup>
  <UseMaui>true</UseMaui>
  <TargetFrameworks>net8.0-ios;net8.0-android;net8.0</TargetFrameworks>
  <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net8.0-windows10.0.19041.0</TargetFrameworks>
  <OutputType>Library</OutputType>
  <ImplicitUsings>enable</ImplicitUsings>
...
```

Adding following XMAL into a library project causes build error:

```
xmlns:ffimageloading="clrnamespace:FFImageLoading.Maui;assembly=FFImageLoading.Maui"
xmlns:fftransformations="clr-namespace:FFImageLoading.Transformations;assembly=FFImageLoading.Maui"
```

```
<ffimageloading:CachedImage
    Aspect="AspectFill"
    DownsampleToViewSize="False"
    FadeAnimationEnabled="True"
    FadeAnimationForCachedImages="True"
    Source="{Binding ImageUri}">
    <ffimageloading:CachedImage.Transformations>
        <fftransformations:GrayscaleTransformation/>
    </ffimageloading:CachedImage.Transformations>
</ffimageloading:CachedImage>
```

### Expected Behavior

Builds successfully

### Actual Behavior

Error	XFC0000	Cannot resolve type "clr-namespace:FFImageLoading.Transformations;assembly=FFImageLoading.Maui:fftransformations:GrayscaleTransformation"

### Basic Information

- Version with issue: 1.2.3
- Last known good version: 
- Platform: Visual Studio 2022 - MAUI project with NET 8

**Comment by microspaze** [comment]
Can you try to build and the demo Sample.csproj project which contains a GrayscaleTransformation in **Sample/Pages/Transformations/GrayscaleTransformationPage.xaml** ? 

Or a **REBUILD** may solve your problem.

**Comment by VegardAaberge** [comment]
I experienced the same issue for TintTransfornation and resolved it by updating Visual Studio from 17.8.3 to 17.9.6.

**Comment by SandipAhluwalia** [comment]
> Can you try to build and the demo **Sample.csproj** project which contains a GrayscaleTransformation in **Sample/Pages/Transformations/GrayscaleTransformationPage.xaml** ?
> 
> Or a **REBUILD** may solve your problem.

Yes, I can build the sample, but that directly references the FFImageLoading.Maui.

Rebuilt my solution including deleting bin and obj folders and the restarting Visual Studio. This made no difference.

Using latest version of Visual Studio 17.9.7

Any other ideas what could be the issue?

**Comment by VegardAaberge** [comment]
@microspaze I experienced the same issue when I added net8.0 as a TargetFramework, which @SandipAhluwalia has included in his TargetFrameworks.

**Sample.csproj** do not include net8.0 as a TargetFramework
`<TargetFrameworks>net8.0-android;net8.0-ios;net8.0-maccatalyst;</TargetFrameworks>`

Try change it to 
`<TargetFrameworks>net8.0;net8.0-android;net8.0-ios;net8.0-maccatalyst;</TargetFrameworks>`

and you will experience the same issue

**Comment by microspaze** [comment]
@VegardAaberge **By default** the transformations are implemented independly by each platforms. They can not be used by **net8.0** targetframework. So you should not add **net8.0** to your project's **TargetFrameworks** list. If you guys have some MUST-NEED scenarios to use it, please tell me. I will try to reimplement those transformations to support pure .net-8.0 framework.

**Comment by VegardAaberge** [comment]
@microspaze Thanks for the prompt response. We do have an important use case for net8.0. Unit test projects target net8.0 and to target the MAUI application, it needs to have net8.0 in the Target Frameworks.

https://learn.microsoft.com/en-us/dotnet/maui/deployment/unit-testing?view=net-maui-8.0

In this scenario, we do not need the actual transformations, but we need it to compile and shouldn't crash when invoked.

**Comment by microspaze** [comment]
@VegardAaberge  I have uploaded a new version of FFImageLoading.Maui (https://www.nuget.org/packages/FFImageLoading.Maui/1.2.6). Please help to verify it for your project. Thanks very much!

**Comment by VegardAaberge** [comment]
@microspaze I have tested in our project and we are now able to target net8.0. Thank you very much!
